### DISCUSSION TITLE
Avoid non-conventional version numbers

### DISCUSSION THREAD

**Discussion by notypecheck** [sqlalchemy/sqlalchemy#9438] [Ideas]
Could we avoid using non-conventional version numbers? For example `2.0.5.post1` instead of `2.0.5` this breaks some tools - currently sentry integration doesn't work because of it and there was a similar problem with `sqlalchemy-utils`.

**Comment by CaselIT** [comment]
Hi,

I think you should report to the tools that breaks with such version. It's a valid version per specification https://peps.python.org/pep-0440/ 

Post was used since there was a problem in wheel creation. From https://peps.python.org/pep-0440/#post-releases

> Some projects use post-releases to address minor errors in a final release that do not affect the distributed software (for example, correcting an error in the release notes).

**Comment by notypecheck** [comment]
Thanks, seems like I have to report it to Sentry then

**Reply by CaselIT** [reply]
In any case we will try to avoid doing post releases if possible. So you could also just skip this version and wait for 2.0.6.

But otherwise tools that interact with pypi versions should support the same numbering as pypi
